tests/test_model.py

```diff
@@ -99,12 +92,12 @@ def test_predict(preprocess_data):
 
 
 def test_load_and_save(learning_task=TASK_CLASSIFICATION):
-    model1 = modelgym.Model(learning_task=learning_task)  # model to save and then read
+    model1 = modelgym.XGBModel(learning_task=learning_task)  # model to save and then read
```

Comment:
why do you test save&load only for XGBModel? shouldn't it work for all?

Reply:
i use xgbmodel because it has more complicated structure neither just Model